### PR TITLE
 JAVA-18118 Verify article Spring REST OAuth2

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/PasswordFlowLiveTest.java [hunk 1]
```diff
@@ -15,33 +15,43 @@
...
+    private final static String CLIENT_ID = "newClient";
+    private final static String CLIENT_SECRET = "newClientSecret";
+    private final static String USERNAME = "john@test.com";
+    private final static String PASSWORD = "123";
+
+    @Test
+    public void givenUser_whenUseFooClient_thenOkForFooResourceOnly() {
+        final String accessToken = obtainAccessToken(CLIENT_ID, USERNAME, PASSWORD);
+
+        final Response fooResponse = RestAssured.given()
```
Comment (dkapil) [inline oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/PasswordFlowLiveTest.java:29]: have me made any changes in this file? if not can we revert it?

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java [hunk 2]
```diff
@@ -16,55 +16,61 @@
...
-	private final static String CLIENT_ID = "newClient";
-	private final static String USERNAME = "john@test.com";
-	private final static String PASSWORD = "123";
-	
-
-	@Test
-	public void givenUser_whenUseFooClient_thenOkForFooResourceOnly() {
-		final String accessToken = obtainAccessToken(CLIENT_ID, USERNAME, PASSWORD);
-
-		final Response fooResponse = RestAssured.given().header("Authorization", "Bearer " + accessToken)
```
Comment (dkapil) [inline oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java:31]: if there are no changes in this file, can we revert this?

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/AuthorizationCodeLiveTest.java [hunk 3]
```diff
@@ -17,63 +17,72 @@
...
-		String tokenUrl = AUTH_SERVER + "/token";
-
-		Map<String, String> loginParams = new HashMap<String, String>();
-		loginParams.put("client_id", CLIENT_ID);
-		loginParams.put("response_type", "code");
-		loginParams.put("redirect_uri", REDIRECT_URL);
-		loginParams.put("scope", "read write");
-
-		// user login
-		Response response = RestAssured.given().formParams(loginParams).get(authorizeUrl);
```
Comment (dkapil) [inline oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/AuthorizationCodeLiveTest.java:48]: let's revert the changes apart from the openid one if we can please?

**Review by dkapil — CHANGES_REQUESTED** [review]

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/PasswordFlowLiveTest.java (hunk 1 above)
Comment (anuragkumawat) [reply oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/PasswordFlowLiveTest.java]: changes done

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/AuthorizationCodeLiveTest.java (hunk 3 above)
Comment (anuragkumawat) [reply oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/AuthorizationCodeLiveTest.java:48]: changes done

File: oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java (hunk 2 above)
Comment (anuragkumawat) [reply oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java]: changes done
